README.md

```diff
@@ -23,8 +23,10 @@ Globally install azure function tools:
 In one terminal run the next development server:
 
 ```bash
-nvm use && yarn dev
+nvm use 14.0.0 && yarn dev
```

Comment:
`nvm use` will use the [.nvmrc](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/blob/main/.nvmrc) file to determine the Node version to use and therefore specifying the version explicitly in the command isn't required.
This is also the case on [line 28](https://github.com/Newarkandsherwood/housing-repairs-online-frontend/pull/19/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R28)

If there is a need to use version 14.0.0 for M1 Macs specifically, then add another section for that.
Although, if that is the case, we should discuss to maintain consistency.

Reply:
Thanks for comment @Bhavesh-Patel ! It seems to be required for M1 Macs (we looked at this with @jamiejordanmt ). I can add a subsection here instead of editing the original instructions but also happy for the discussion.

@maysakanoni have you had a chance to try it on yours? I remember you've mentioned having the same issue.

Reply:
I've moved this to the bottom section. I'm also happy to discuss :-).